artifacts/rhdh-plugins-reference/nexus-repository-manager/nexus-repository-manager-plugin-readme.adoc

```diff
@@ -88,7 +88,7 @@ The Nexus Repository Manager is a front-end plugin that enables you to view the
 .Prerequisites
 
 - Your {product-short} application is installed and running.
-- You have installed the Nexus Repository Manager plugin. For the installation process, see <<Installing and configuring the Nexus Repository Manager plugin>>.
+- You have installed the Nexus Repository Manager plugin. For the installation process, see documentation on Installing and configuring the Nexus Repository Manager plugin.
```

Comment:
If we’re including the statement, could you please clarify why the link is being removed?